ts/RailroadMap.ts

```diff
@@ -1304,7 +1313,7 @@ export class RailroadMap {
                 .text((block) => block
                     .text(str)
                     .dx(300))
-                .attr('transform', makeTransformT(cp0, cp1))
+                .attr('transform', makeTransformT(cp0, cp1)) // embix: how?
```

Comment:
I recommend updating the `makeTransformT` or underlying `makeTransformF` function. You'll probably need to pass invert to it, and then you can add 180 to the angle if it's false.

Reply:
I considered introducing a makeTransform overload that but rejected the idea as it could turn into a time sink. Due to lots of uses, a bug would be hard to discover, even harder to debug. If there were automated regression test cases, I'd have given it a shot.

Reply:
I'm not sure what you mean by a time sink. While implementing the change may require some effort, it is the correct approach to ensure these functions handle text orientation properly. Since their purpose is to display text in the correct orientation, updating them to support this functionality is essential.